tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -1,5 +1,18 @@
 [assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/DataDog/dd-trace-dotnet.git")]
 
+namespace Datadog.Trace.Ci.Coverage.Attributes
+{
+    public class AvoidCoverageAttribute : System.Attribute
+    {
+        public AvoidCoverageAttribute() { }
+    }
+    public class CoveredAssemblyAttribute : System.Attribute
+    {
+        public CoveredAssemblyAttribute(ulong totalMethods, ulong totalInstructions) { }
+        public ulong TotalInstructions { get; }
+        public ulong TotalMethods { get; }
+    }
```

Comment:
could/should these be `internal`?

Reply:
I can't switch to internal here, both attributes will be injected on the target assembly at assembly level, and we need to pass the visibility checks, for other types we have a special attribute to bypass the checks, but it doesn't work on assembly attributes.